branchinfo.txt

```diff
@@ -1,9 +1,15 @@
 # This is a file containing environment variables specific to this branch
 # Any line that is not blank and does not start with '#' is interpreted as a variable to set
 # Each line is expected to be in the format "[Name]=[Value]".
+#
+#
+# !!! WARNING !!! WARNING !!! WARNING !!! WARNING !!! WARNING !!! WARNING
+#       "master" branch temporarily produced 1.0.1 builds. The value here
```

Comment:
What's the plan here going forward?  What if we need to service the Microsoft.NETCore.App package?  What version will we use?